install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -668,6 +666,30 @@ spec:
             done
         terminationMessagePolicy: FallbackToLogsOnError
       {{- end }} # wait-for-kube-proxy
+      # Install the CNI binaries in an InitContainer so we don't have a writable host mount in the agent
+      - name: install-cni-binaries
+        image: {{ include "cilium.image" .Values.image | quote }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        command:
+          - "/install-plugin.sh"
+        resources:
+          requests:
+            cpu: 100m
+            memory: 10Mi
+        securityContext:
+          {{- if not .Values.securityContext.privileged }}
```

Comment:
Is there a missing else branch here? What's supposed to be here if `.Values.securityContext.privileged` is true?

Reply:
Makes sense, thanks for the explanation!